src/lib.rs

```diff
@@ -1035,16 +1038,136 @@ impl<T: Clone + Integer + Signed> Signed for Ratio<T> {
 }
 
 // String conversions
-impl<T> fmt::Display for Ratio<T>
+impl<T> Display for Ratio<T>
 where
-    T: fmt::Display + Eq + One,
+    T: Display + Clone + Integer,
```

Comment:
breaking change!

Reply:
It doesn't have to be breaking though. We could leave the old implementation of Display and merge the other formatting traits. Then, at some version where we're making breaking changes, we could merge a PR to make Display act like the new formatting Traits (basically support padding)